packages/components/src/components/form-progress/form-progress.jsx

```diff
@@ -42,16 +44,31 @@ class FormProgress extends React.PureComponent {
                             }}
                         />
                         {steps.map((item, idx) => (
-                            <div
-                                key={idx + 1}
-                                className={classNames('dc-form-progress__step', {
-                                    'dc-form-progress__step--active': idx <= current_step,
-                                    'dc-form-progress__step--current': idx === current_step,
-                                })}
-                            >
-                                <span className='identifier'>{idx + 1}</span>
-                                <p className='title'>{item.header.title}</p>
-                            </div>
+                            <>
```

Comment:
```suggestion
                            <React.Fragment>
```

Reply:
Is it necessary to use the longer syntax while there is not any key or attribute?

Reply:
It's our project-wide convention.